packages/gateway/src/features/registration/fhir-builders.ts

```diff
@@ -3149,24 +2823,22 @@ export const builders: IFieldBuilders = {
         if (!certDocResource.extension) {
           certDocResource.extension = []
         }
-        const hasVerifiedExt = certDocResource.extension.find(
-          (extention) => extention.url === HAS_SHOWED_VERIFIED_DOCUMENT
+        const hasVerifiedExt = findExtension(
+          HAS_SHOWED_VERIFIED_DOCUMENT,
+          certDocResource.extension
         )
+
         if (!hasVerifiedExt) {
           certDocResource.extension.push({
             url: `${OPENCRVS_SPECIFICATION_URL}extension/hasShowedVerifiedDocument`,
-            valueString: fieldValue
+            valueBoolean: fieldValue
           })
         } else {
-          hasVerifiedExt.valueString = fieldValue
+          hasVerifiedExt.valueBoolean = fieldValue
```

Comment:
Are we handling this change in the typeResolver as otherwise this would become a breaking one

Reply:
We are 👍 https://github.com/opencrvs/opencrvs-core/pull/5877/files#diff-a1167432e4624234c311aef46e5b06174279a607f9a10c2947281a660288e2b1L1137-L1143